libsolidity/parsing/Parser.cpp

```diff
@@ -307,6 +307,24 @@ Declaration::Visibility Parser::parseVisibilitySpecifier(Token::Value _token)
 	return visibility;
 }
 
+string Parser::visibilitySpecifierName(Declaration::Visibility _visibility)
```

Comment:
What about making that a function `toString`, declared closer to `Declaration::Visibility`? I think we use that also in other places.